src/test/java/org/jabref/logic/formatter/bibtexfields/RemoveHyphenatedNewlinesFormatterTest.java

```diff
@@ -15,8 +15,7 @@ public void setUp() {
 
     @Test
     public void removeHyphensBeforeNewlines() {
-        assertEquals("water", formatter.format("wa-\nter"));
-        assertEquals("water", formatter.format("wa-\r\nter"));
-        assertEquals("water", formatter.format("wa-\rter"));
+        String newLine = String.format("%n");
```

Comment:
Please keep these tests (here and below). The text that we want to format might contain linux line endings on Windows and vice versa. But you can of course also add a tests for ̀ %n`. 
Please change also all regex patterns to use `\R` instead of `%n` for newlines.

Reply:
Finally leaving them untouched. 

In summary, Pattern.compile("...") changed from `(\r?\n|\r)` to `\\R`